### ISSUE TITLE
Annotation window covers the bottom of the iframe view

### ISSUE THREAD

**Issue by acarnwath** [pattern-lab/patternlab-node#576]
I am using Pattern Lab Node `v2.6.2` on `Mac`, with Node `v4.2.4`, using the `Gulp` Edition.

##### Expected Behavior
When I have a long pattern that causes the window to scroll, and an annotation is at the bottom of it, when I open up the annotation window, I should be able to see what the annotation is referring to.

##### Actual Behavior
The annotation overlays the iframe and does not adjust its height, so you can never scroll to see the bottom of the page with the annotation overlay open.

![annotation-closed](https://cloud.githubusercontent.com/assets/3651849/21273083/f6dbec78-c38f-11e6-98c6-5439efc26ac4.png)
This button has an annotation on it. This screenshot is scrolled all the way down - can't scroll any more.

![annotation-open](https://cloud.githubusercontent.com/assets/3651849/21273082/f6da3658-c38f-11e6-9fee-c24d43a83eaa.png)
The annotation is highlighted for that button, but you can't see the button because it's underneath the modal. This screenshot is scrolled all the way down - can't scroll any more.

##### Possible Solution
In patternlab-viewer.js, adjust the bottom of `$('#sg-vp-wrap')` (and possibly add matching CSS transition) to match the position used here:
```
  /**
  * slides the modal window into or out of view
  * @param  {Integer}      where the modal window should be slide to
  */
  slide: function(pos) {
    pos = (pos === 0) ? 0 : -pos;
    $('#sg-modal-container').css('bottom',pos);
  },
```

**Comment by acarnwath** [comment]
I fixed it temporarily by adding the following in source/_meta/_01-foot.mustache (note that I have included jQuery prior, this does not come out of the box with PL.

```
<script>

    	// Manipulating the height of the iframe so that the annotations don't cover up content
        var $modal = $(parent.document).find('#sg-modal-container'),
        	$iframeWrap = $(parent.document).find('#sg-vp-wrap');

		$modal.on('webkitTransitionEnd otransitionend oTransitionEnd msTransitionEnd transitionend', function(e) {
			var newBottom = $modal.height() + parseInt($modal.css('bottom'),10);
			newBottom = newBottom < 0 ? 0 : newBottom;
			$iframeWrap.css('bottom', newBottom);
		});

    </script>
```

**Comment by bmuenzenmeyer** [comment]
I think the proper place to fix this is https://github.com/pattern-lab/styleguidekit-assets-default/issues/6 (which I just reopened) but I will most certainly reference your work when we get to it - thanks @acarnwath !

**Comment by frostyweather** [comment]
This has been addressed in #6
